test/nodesecure.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, after, describe, it } from "node:test";
+import assert from "node:assert";
+
+// Import Third-party Dependencies
+import { execute } from "../src/plugins/nodesecure";
+
+
+import { MockAgent, setGlobalDispatcher } from "undici";
+
+const kApiUrl = "https://api.securityscorecards.dev";
+
+describe("execute()", async() => {
+  const mockAgent = new MockAgent();
+  mockAgent.disableNetConnect();
+
+  const mockPool = mockAgent.get(kApiUrl);
+
+  setGlobalDispatcher(mockAgent);
+
+  after(async() => {
+    await mockAgent.close();
+  });
+
+  test("should return void", async() => {
```

Comment:
```suggestion
  it("should return void", async() => {
```

I think we can use only `it` and remove `test` from the imports ?